ceph/rbd/workflows/snap_clone_operations.py

```diff
@@ -565,11 +583,11 @@ def wrapper_for_image_snap_ops(**kw):
             "test_ops_parallely": <>
         }
     """
-    rbd = kw.get("rbd")
-    pool = kw.get("pool")
+    rbd = kw.pop("rbd")
```

Comment:
why `pop` here?

Reply:
pop to avoid duplicate kw arg error, same like wrapper_for_image_ops